geoips/xarray_utils/data.py

```diff
@@ -687,12 +753,14 @@ def sector_xarrays(
 
         if is_dynamic_sector(area_def):
             LOG.info(
-                "  Trying to sector %s with dynamic time %s, data time %s to %s, %s points",
-                area_def.area_id,
-                area_def.sector_start_datetime,
-                xobj.start_datetime,
-                xobj.end_datetime,
-                xobj["latitude"].size,
+                """  Trying to sector {0} with dynamic time {1}, data time {2} to {3},
```

Comment:
Another one that should probably be 
"line1"
"line2"

Reply:
updated this